test/helper.rb

```diff
@@ -81,7 +81,10 @@ def assert_parser_unchanged(asset)
     end
 
     def assert_output_unchanged(file, args, actual = nil)
-      actual, args = args, nil if actual == nil
+      if actual.nil?
+        actual = args
+        args = nil
+      end
```

Comment:
I don't like this "correction". Which cop is this?

(The original code would have been better with `actual.nil?` or `unless actual`, rather than `actual == nil`, though.)